UnityProject/Assets/Scripts/PlayGroups/PlayerNetworkActions.cs

```diff
@@ -448,8 +458,7 @@ public void RpcSpawnGhost()
 			Camera.main.cullingMask &= ~(1 << LayerMask.NameToLayer("FieldOfView"));

```

Comment:
This is where you should set the Camera2D followcontrol dampening property, not from the closet update loop

Reply:
Seems like he did :)